JumpscaleLibs/tools/tf_gateway/TFGateway.py

```diff
@@ -217,6 +220,165 @@ def domain_register_srv(self, name, domain, host, port, priority=10, weight=100)
             name, domain, "srv", records=[{"host": host, "port": port, "priority": priority, "weight": weight}]
         )
 
+    def domain_unregister(self, name, domain="bots.grid.tf.", record_type="a", prefix=DNS_PREFIX, records=[]):
```

Comment:
dns prefix comment and NEVER pass list in params as a default value, please use None and in the function code do
```python
records = records or []
```